gtk/gtk.go

```diff
@@ -1372,14 +1372,18 @@ func (v *Window) SetTypeHint(hint gdk.WindowTypeHint) {
 // gtk_window_set_frame_dimensions //deprecated since 2.24
 // gtk_window_set_has_frame  //deprecated since 2.24
 // gtk_window_set_mnemonic_modifier
-// gtk_window_set_skip_taskbar_hint
 // gtk_window_set_skip_pager_hint
 // gtk_window_set_urgency_hint
 
 func (v *Window) SetAcceptFocus(setting bool) {
 	C.gtk_window_set_accept_focus(WINDOW(v), gbool(setting))
 }
 
+func (v *Window) SetSkipTaskbarHint(setting bool) {
+	C.gtk_window_set_skip_taskbar_hint(WINDOW(v), gbool(setting))
+
```

Comment:
It is done in the next commit(https://github.com/mattn/go-gtk/pull/301/commits/f9becbde2a7128bd08a5fb0c66e551e2f9cf3004).